src/pages/index.tsx

```diff
@@ -1,19 +1,23 @@
 import Head from 'next/head'
 import Image from 'next/image'
 import { Roboto } from 'next/font/google'
-import { lorem }from '../constants/lorem'
-import { Box, Typography } from '@mui/material'
-import { colors, mediaQueriesThresholds } from '../constants/constants.js'
+import { missionStatement, keyPoints }from '../constants/strings'
```

Comment:
You should standardize your path accessing. Either replace .. navigation with @ or replace @ with ..